modeldb/modeldb-run.yaml

```
@@ -692,15 +705,12 @@
     - run()
     - verify_graph_()
 105507:
-    curate_patterns:
-    - pattern: '^([0-9.\-e]+) kilocycles$'
-      repl: '%some_count% kilocycles'
     run:
     - load_file("batch_.hoc")
     - verify_graph_()
     script:
     # 424242: this is just avoiding a timing-related number being printed
-    - sed -i'.bak' -e 's/batch_flag=0/batch_flag=1/g;s/tstop = 1e3/tstop = 20/g;s#return tti/1e3#return 424242#g' batch_.hoc
+    - sed -i.bak  -e 's/batch_flag=0/batch_flag=1/g;s/tstop = 1e3/tstop = 20/g;s#return tti/1e3#return 424242#g;s#print tti/.*#print "%some_time%"#g' batch_.hoc
```

I remember the only pattern that was portable on macOS was like
```suggestion
    - sed -i'.bak' -e 's/batch_flag=0/batch_flag=1/g;s/tstop = 1e3/tstop = 20/g;s#return tti/1e3#return 424242#g;s#print tti/.*#print "%some_time%"#g' batch_.hoc
```